drivers/nrf24l01/nrf24l01test.py

```diff
@@ -69,7 +101,6 @@ def slave():
     print('NRF24L01 slave mode, waiting for packets... (ctrl-C to stop)')
 
     while True:
-        machine.idle()
```

Comment:
Did you ever get to the bottom of the issues with this on pyboard?

Reply:
No, this has me stumped I'm afraid.

Reply:
Ok, no worries, it's just a test anyway so not critical to have this line.